src/components/artwork/fillwidth_item.tsx

```diff
@@ -83,11 +80,11 @@ export default Relay.createContainer(StyledFillwidthItem, {
 
 interface RelayProps {
   artwork: {
-    href: string | null,
+    href: string | null
     image: {
-      placeholder: number | null,
-      url: string | null,
-      aspect_ratio: number | null,
-    } | null,
-  },
+      placeholder: number | null
```

Comment:
Yeah, we had that discussion in the Emission PR, we're good with this 👍 

It doesn't change behaviour, Microsoft use semi-colons here.